.travis.yml

```diff
@@ -34,19 +35,23 @@ jobs:
   allow_failures:
     - script: yarn test:saucelabs
     - script: yarn test:saucelabs --database-firestore-only
+    - script: node scripts/testing/database-emulator-test.js
```

Comment:
FWIW it could be helpful to document these with comments to explain the current state and future plans.  e.g.:

```
# TODO(yifanyang): Once we verify the emulator tests are reliable, we should make these tests blocking rather than allow failures.
```

Ideally we'd comment the saucelabs ones too, but that's orthogonal to this PR, so feel free to leave as-is for now.

Also, you added names for the ones below.  Should these have names too?

Reply:
Done for adding comments.

Names for the below ones will actually display on Travis run page. But names here will have no use. And they can actually be different from the ones below, and everything will still work. Travis is only using the 'scirpt' content here to identify different jobs. So I feel like there's not much value for adding names here.